src/index.rs

```diff
@@ -369,7 +370,11 @@ pub async fn sync_indexes<CollConf: CollectionConfig>(
             let mut to_drop = Vec::new();
             for (i, index) in indexes.0.clone().into_iter().enumerate() {
                 let mut text_index_keys = None;
-                let index_doc = if index.keys.iter().any(|ind| matches!(ind, IndexKey::TextIndex(_))) {
+                let index_doc = if index
+                    .keys
+                    .iter()
+                    .any(|ind| matches!(ind, IndexKey::TextIndex(_)))
+                {
```

Comment:
question: Do you know why the formatting was changed here? Is it the result of `cargo fmt` using the stable channel? I see other similar places.

Reply:
The formatting has been changed due to cargo. 
I just ran cargo fmt as simple as that and it reformats it that way automatically
```shell
cargo fmt --verbose 
```

Reply:
Sounds good. I just realized that our CI was dead and we were not checking the formatting anymore.

Reply:
I noticed the CI was dead as well. We can adress it from another PR or this one?

Reply:
I opened a PR for that already: https://github.com/Devolutions/mongodm-rs/pull/31